src/main/java/org/folio/circulation/domain/Request.java

```diff
@@ -347,8 +347,13 @@ public String getPatronComments() {
   public Request truncateRequestExpirationDateToTheEndOfTheDay(DateTimeZone zone) {
     DateTime requestExpirationDate = getRequestExpirationDate();
     if (requestExpirationDate != null) {
-      final DateTime dateTime = atEndOfDay(requestExpirationDate, zone);
-      write(requestRepresentation, REQUEST_EXPIRATION_DATE, dateTime);
+      // TODO: this introduces behavioral change not yet intended, use this after converting JodaTime to JavaTime.
+      //final DateTime dateTime = atEndOfDay(requestExpirationDate, zone);
+      //write(requestRepresentation, REQUEST_EXPIRATION_DATE, dateTime);
```

Comment:
What behaviour change does this introduce?

Reply:
```
Is it to do with how the end of day is interpreted relative to the zone?
```
Yes.

Reply:
Ok, thanks.